pkg/models/querybuilder_scene.go

```diff
@@ -191,6 +192,14 @@ func (qb *SceneQueryBuilder) Count() (int, error) {
 	return runCountQuery(buildCountQuery("SELECT scenes.id FROM scenes"), nil)
 }
 
+func (qb *SceneQueryBuilder) SizeCount() (string, error) {
+	sum, err := runSumQuery("SELECT SUM(size) as sum FROM scenes", nil)
+	if err != nil {
+		return "O", err
```

Comment:
Out of curiosity, what's the purpose of returning O? Can you not return an empty string?

Reply:
0 seemed better than an empty string since we are talking about size. But doesn't actually matter , do you think `""` would look better?

Reply:
I don't really have a preference since it's an error anyway, I was just curious since it returns the letter O. 0 makes more sense.

Reply:
oh it was supposed to be "0" (zero) ! Just noticed that now , typo...